.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [6.x, 8.x, 9.x, 10.x, 12.x, 13.x]
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install
+        run: |
+          npm install
+      - name: Run Tests
```

Comment:
Maybe, can be good add the lint check

Reply:
the `test` run has the `standard` check already :+1: